src/style.css

```
@@ -1 +1,68 @@
-/* Styles go here */
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
+  max-height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo_link {
+  display: inline-block;
+  margin: 10px 50px;
+  height: 40px;
+}
```

You already set 50px padding for the `header`, so you don't need to set horizontal margins here
```suggestion
.logo_link {
  display: inline-block;
  margin-top: 10px;
  height: 40px;
}
```